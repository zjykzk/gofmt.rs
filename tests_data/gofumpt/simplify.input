package p

const ()

const (
// Comment
)

type ()

type (
// Comment
)

var ()

var (
// Comment
)

var _ = [][]int{[]int{1}}
