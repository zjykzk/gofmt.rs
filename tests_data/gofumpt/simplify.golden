package p

const (
// Comment
)

type (
// Comment
)

var (
// Comment
)

var _ = [][]int{{1}}
